TEveEventDisplay/src/TEveMu2eDataInterface.cc

```diff
@@ -376,43 +377,59 @@ template <typename L> void maxminCRV(L data, double &max, double &min){
       TXYMgr->ImportElements(fTrackList2D, scene1); 
       TRZMgr->ImportElements(fTrackList2D, scene2); 
       if(seedcol!=0){  
-        for(unsigned int k = 0; k < seedcol->size(); k = k + 20){
+        for(unsigned int k = 0; k < seedcol->size(); k = k + 20){   
+       
           KalSeed kseed = (*seedcol)[k];
-          TEveMu2eCustomHelix *line = new TEveMu2eCustomHelix();
-          TEveMu2eCustomHelix *line_twoD = new TEveMu2eCustomHelix();
-          line->fKalSeed = kseed;
-          line->SetSeedInfo(kseed);
+          const std::vector<mu2e::KalSegment> &segments = kseed.segments();
+      size_t nSegments=segments.size();
+      std::cout<<"KalSegment size ="<<segments.size()<<std::endl;
+      if(nSegments==0) continue;
+      const mu2e::KalSegment &segmentFirst = kseed.segments().front();
+      const mu2e::KalSegment &segmentLast = kseed.segments().back();
+      double fltLMin=segmentFirst.fmin();
+      double fltLMax=segmentLast.fmax();
+      std::cout<<"f min ="<<fltLMin<<"f max ="<<fltLMax<<std::endl;
+
+      TEveMu2eCustomHelix *line = new TEveMu2eCustomHelix();
+      TEveMu2eCustomHelix *line_twoD = new TEveMu2eCustomHelix();
+
+      line->fKalSeed = kseed;
+      line->SetSeedInfo(kseed);
+  
+      for(size_t m=0; m<nSegments; m++)
+      {
+        const mu2e::KalSegment &segment = segments.at(m);
+
+        fltLMin=segment.fmin();
+        fltLMax=segment.fmax();
+        std::cout<<"Inside loop , fmin ="<<fltLMin<<" fmax="<<fltLMax<<std::endl;
```

Comment:
this looks like a debugging statement.  It should be put in an if(debug_){   clause.

Reply:
Yes, thank you. I think I have taken care of it now.